src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify the code flow. Since you're returning `totalCost` at the end, you can simply return the discounted cost directly within the `if` blocks and eliminate the need for an `else if` structure.